lbrynet/tests/unit/core/test_Wallet.py

```diff
@@ -217,7 +217,7 @@ def update_balance():
         d.addCallback(lambda _: self.assertEqual(5, wallet.get_balance()))
         d.addCallback(lambda _: wallet.reserve_points('testid', 2))
         d.addCallback(lambda _: wallet.claim_name('test', 4, test_claim_dict))
-        self.assertFailure(d, InsufficientFundsError)
+        self.assertFailure(d, Exception)
```

Comment:
Testing for `Exception` seems a bit dangerous. This would pass the test on any exception, even if it has nothing to do with the error being tested.

Reply:
This is what is returned from lbryum I guess, maybe I need to look at the lbryum error messages too.
Also would it be bad practice if I check the error message string and then decide which exception to raise?? Something like
```python
if claim["msg"] == "Insufficient funds":
  Raise InsufficientFundsError()
```
here claim is the JSON signature returned from `lbryum`

Reply:
Checking for an error condition in a response from lbryum and throwing an appropriate exception makes sense to me. Although lbryum is not consistent about the insufficient funds errors, maybe we need to add error codes first as @kauffj suggested in another issue.

Reply:
Yes, the function for checking the above stuff in lbryum is a bit convoluted, with many abstract returns. Maybe I'll check for error message in daemon and raise specific Exceptions for possible cases, and a different but non-generic and descriptive errors for the rest of the cases.